topicgenerator/results/generated-topics_GPT_3_5_TURBO_16K_20231113T135602Z.csv

```diff
@@ -1,24 +1,24 @@
+"#workflow/github.com/DataBiosphere/topmed-workflows/UM_variant_caller_wdl",1.32.0,https://raw.githubusercontent.com/DataBiosphere/topmed-workflows/1.32.0/variant-caller/variant-caller-wdl/topmed_freeze3_calling.wdl,false,14201,22,stop,"Run TopMed variant calling workflow using Michigan's pipeline with Docker images, file sizes calculations, and chromosome processing."
+"#workflow/github.com/DataBiosphere/analysis_pipeline_WDL/vcf-to-gds-wdl",v7.1.1,https://raw.githubusercontent.com/DataBiosphere/analysis_pipeline_WDL/v7.1.1/vcf-to-gds/vcf-to-gds.wdl,false,2631,25,stop,"Convert VCF files to GDS files, uniquely identify variant IDs, and check GDS files against their VCF inputs."
+"#workflow/github.com/DataBiosphere/analysis_pipeline_WDL/ld-pruning-wdl",v7.1.1,https://raw.githubusercontent.com/DataBiosphere/analysis_pipeline_WDL/v7.1.1/ld-pruning/ld-pruning.wdl,false,3586,24,stop,"Execute ldpruning workflow on multiple GDS files with optional parameters for pruning and merging, generating output for downstream analysis."
+"#workflow/github.com/AnalysisCommons/genesis_wdl/genesis_GWAS",v1_5,https://raw.githubusercontent.com/AnalysisCommons/genesis_wdl/v1_5//genesis_GWAS.wdl,false,3542,18,stop,Performing genetic association testing and summarization of results using GENESIS in a comprehensive workflow.
+"#workflow/github.com/aofarrel/covstats-wdl",master,https://raw.githubusercontent.com/aofarrel/covstats-wdl/master/covstats/covstats.wdl,false,2187,29,stop,"Extract read length, coverage, and filenames from input BAMs or CRAMs, summarize them, and generate a report in a Python script."
+"#workflow/github.com/broadinstitute/warp/Optimus",aa-PD2413,https://raw.githubusercontent.com/broadinstitute/warp/aa-PD2413/pipelines/skylab/optimus/Optimus.wdl,false,2468,27,stop,"Implement a workflow to process 10x genomics data including correction, alignment, marking duplicates, and producing BAM and sparse matrix files."
+"#workflow/github.com/theiagen/terra_utilities/Concatenate_Column_Content",v1.4.1,https://raw.githubusercontent.com/theiagen/terra_utilities/v1.4.1/workflows/wf_cat_column.wdl,false,173,9,stop,Concatenate files and capture version for analysis.
+"#workflow/github.com/gatk-workflows/seq-format-conversion/BAM-to-Unmapped-BAM",3.0.0,https://raw.githubusercontent.com/gatk-workflows/seq-format-conversion/3.0.0//bam-to-unmapped-bams.wdl,false,996,28,stop,"Convert BAM to unmapped BAMs, reverting and sorting them using GATK in optimized runtime parameters on Broad's Google Cloud Platform."
+"#notebook/github.com/denis-yuen/test-notebooks/ibm-tax-maps",0.2,https://raw.githubusercontent.com/denis-yuen/test-notebooks/0.2/ibm-et/jupyter-samples/tax-maps/Interactive_Data_Maps.ipynb,true,16283,24,stop,"Summarize the notebook: ""Plotting tax collections by state for different categories using interactive data maps and data analysis."""

```

Comment:
Strange that it starts with "summarize the notebook"

Reply:
But otherwise cool, especially since the description in the ipynb is bracketed by a lot of json-ish stuff